package.json

```diff
@@ -38,8 +38,6 @@
     "bump:playground:version": "cd playground && npm --no-git-tag-version --allow-same-version version $VERSION",
     "bump:webcomp:version": "cd web-component && npm --no-git-tag-version --allow-same-version version $VERSION",
     "bump:lib:version": "cd library && npm --no-git-tag-version --allow-same-version version $VERSION",
-    "bump:version": "npm run bump:lib:version && npm run bump:webcomp:version && npm run bump:playground:version && npm run install:reactcomp:webcomponent && npm run install:reactcomp:playground",
```

Comment:
not the whole script, just `npm run install:reactcomp:playground` please, and `npm run bump:playground:version` as we still need for others

Reply:
I mean we still need it but as 

```
"bump:version": "npm run bump:lib:version && npm run bump:webcomp:version && npm run install:reactcomp:webcomponent",
```